packages/desktop/src/webview.rs

```diff
@@ -22,16 +22,19 @@ pub struct WebviewInstance {
     // Wry assumes the webcontext is alive for the lifetime of the webview.
     // We need to keep the webcontext alive, otherwise the webview will crash
     _web_context: WebContext,
+
+    // Same with the menu.
+    // Currently it's a box<dyn any> because 1) we don't touch it and 2) we support a number of platforms
```

Comment:
We could change this to a type alias exported from `impl_` to avoid the any once we need access to the menu to support custom menus defined in components/hooks